db/compaction/compaction_job.h

```diff
@@ -299,6 +319,18 @@ class CompactionJob {
   BlobFileCompletionCallback* blob_callback_;
 
   uint64_t GetCompactionId(SubcompactionState* sub_compact) const;
+  // Stores the number of reserved threads in shared env_ for the number of
+  // extra subcompaction in kRoundRobin compaction priority
+  int extra_num_subcompaction_threads_reserved_;
+  // Stores the number of available compactions allowed against maximum
+  // compaction limit in kRoundRobin compaction priority. This would be an upper
+  // bound of reserved threads.
+  int available_bg_compactions_db_limit_;
```

Comment:
Wouldn't hurt to add one more prep `int available_bg_compactions_against_db_limit_;` to ensure people w/o reading comments do not misunderstand this variable as a "limit"

Reply:
Sure. Will do.